win/CS/HandBrakeWPF/Views/QueueView.xaml

```diff
@@ -436,11 +438,14 @@
                                     <TextBlock FontWeight="Bold" Text="Status:" VerticalAlignment="Top"  Grid.Row="0" />
                                     <TextBlock Text="{Binding SelectedTask.JobProgress.JobStatus}" TextWrapping="WrapWithOverflow" VerticalAlignment="Top" TextTrimming="CharacterEllipsis" Grid.Row="0" Grid.Column="1" />
 
-                                    <TextBlock FontWeight="Bold" Text="{x:Static Properties:Resources.QueueView_Source}" VerticalAlignment="Top" Grid.Row="1" Grid.Column="0" />
-                                    <TextBlock Text="{Binding SelectedTask.Task.Source}" TextWrapping="Wrap" Grid.Row="1" Grid.Column="1" />
+                                    <TextBlock FontWeight="Bold" VerticalAlignment="Top" Text="{x:Static Properties:Resources.QueueView_SelectedPresets}" Grid.Row="1" Grid.Column="0"   />
+                                    <TextBlock Text="{Binding SelectedTask.SelectedPresetKey, Converter={StaticResource presetConverter}}" Grid.Row="1" Grid.Column="1" />

```

Comment:
It might be better to place this just before the "Picture Settings".  That keeps it such that the source information is together and the task information is together.

Reply:
Okay

Reply:
Is this Okay?
![image](https://user-images.githubusercontent.com/51527710/130370384-02a80423-105f-4600-9977-66861c594dd0.png)

Reply:
yup, looks good.